src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```
@@ -39,5 +39,12 @@
     <None Include="..\lib\Microsoft.SqlServer.TransactSql.ScriptDom.dll" Pack="true" PackagePath="lib/$(TargetFramework)" />
     <None Include="..\README.md" Pack="true" PackagePath="" />
   </ItemGroup>
-
+  <Target Name="CopyNupkg" AfterTargets="Build">
+    <ItemGroup>
+      <_Packages Include=".\bin\$(Configuration)\*.*" />
+      <_Packages Include=".\obj\$(Configuration)\*.*" />
```

You should only copy the nuget packages, not all files (and especially not ones from the obj folder, that doesn't have anything we need)

```suggestion
      <_Packages Include=".\bin\$(Configuration)\*.nupkg" />
```